mkdocs.yml

```diff
@@ -77,7 +77,12 @@ nav:
         # - Community meeting (Sept'22): meetings/2022-09-amsterdam.md
     - Contact info: contact.md
   - Systems where EESSI is available: systems.md
-  - Governance: governance.md
+  - Governance:
+    - Charter: governance/charter.md
+    - Governance: governance/governance.md
+    - EESSI Policies: governance/policies.md
+    - Code of Conduct: governance/code_of_conduct.md
+    - Current Steering Committee: governance/steering_committee.md
```

Comment:
We should either set up an auto-redirect for `eessi.io/docs/governance` to the moved `steering_comittee` URL, or set up a simple `index.md` in `/governance` with bullets that points to the subpages.